Cargo.toml

```diff
@@ -14,14 +14,15 @@ categories = ["web-programming"]
 [dependencies]
 html5ever = "0.25"
 scraper = "0.12"
-reqwest = "0.11"
+reqwest = "0.11.10"
```

Comment:
```suggestion
reqwest = "0.11"
```

Reply:
It doesn't make a difference for my use case. But why not update it?